mm2src/coins/utxo/utxo_common.rs

```diff
@@ -3215,47 +3279,72 @@ pub fn dex_fee_script(uuid: [u8; 16], time_lock: u32, watcher_pub: &Public, send
         .into_script()
 }
 
-pub async fn list_unspent_ordered<'a, T: UtxoCommonOps>(
+pub async fn get_unspent_ordered_list<'a, T: ListUtxoOps>(
     coin: &'a T,
     address: &Address,
-) -> UtxoRpcResult<(Vec<UnspentInfo>, AsyncMutexGuard<'a, RecentlySpentOutPoints>)> {
+) -> UtxoRpcResult<(Vec<UnspentInfo>, RecentlySpentOutPointsGuard<'a>)> {
+    let (mut unspent_map, recently_spent) = coin.get_unspent_ordered_map(vec![address.clone()]).await?;
+    let unspents = unspent_map.remove(address).or_mm_err(|| {
+        let error = format!("'get_unspent_ordered_map' should have returned '{}'", address);
```

Comment:
Batch seems to be not required for the single address request.

Reply:
Fixed

Reply:
I shared my thoughts here https://github.com/KomodoPlatform/atomicDEX-API/pull/1255#discussion_r857362577
I hope it's an answer to this question.

Reply:
Ok, it's acceptable if we won't have too many of these possible `None` handlers.

Reply:
I've changed this, so now `get_X_unspents_list` uses single RPC request and doesn't rely on similar `get_X_unspents_map`.